### PR TITLE
DEP-156: Tidy up accessibility for webchat page

### PR DESCRIPTION
**Pull request by tonymyers**

### CODE REVIEW

File: app/views/main.scala.html [hunk 1]
```diff
@@ -17,8 +17,7 @@
 @import config.ApplicationConfig
 @(title: String,
   mainContentHeader: Html = HtmlFormat.empty,
-  appConfig: config.AppConfig = config.ApplicationConfig,
-  bodyClass: Option[String] = None)(mainContent: Html)(implicit request: Request[_], lang : Lang, messages: Messages)
```
Comment (oc-ste) [inline app/views/main.scala.html]: This should not have been removed, you now can't change bodyClass as its now set to None below, was this intentional?

File: conf/messages [hunk 2]
```diff
@@ -200,3 +200,63 @@ language-switcher.welsh=Cymraeg
...
 
 help.webchat.title=Accessibility for webchat
+help.webchat.askhmrcwebchat.p1=The Ask HMRC webchat is a service that enables you to communicate with an HMRC adviser over the internet.
+help.webchat.askhmrcwebchat.p2=Ask HMRC webchat is built and run by Nuance, a provider of customer engagement software. We want as many people as possible to be able to use this website and webchat. For example, that means you should be able to:
+help.webchat.askhmrcwebchat.li1=zoom in up to 200% without the text spilling off the screen
+help.webchat.askhmrcwebchat.li2=navigate most of the webchat using just a keyboard
+help.webchat.askhmrcwebchat.li3=navigate most of the webchat using speech recognition software
+help.webchat.askhmrcwebchat.li4=listen to most of the webchat using a screen reader (including the most recent versions of JAWS, NVDA and VoiceOver)
+help.webchat.askhmrcwebchat.p3={0} has advice on making your device easier to use if you have a disability.
+help.webchat.askhmrcwebchat.url=https://mcmw.abilitynet.org.uk/
```
Comment (oc-ste) [inline conf/messages]: URLs should be in an app config not in messages

File: conf/messages [hunk 3]
```diff
@@ -200,3 +200,63 @@ language-switcher.welsh=Cymraeg
...
+help.webchat.howaccessible.li6=Users may find that when the chat is finished, they cannot tab backwards through the chat
+help.webchat.howaccessible.li7=To find the privacy and accessibility links, users should tab beyond the message input box
+help.webchat.howaccessible.li8=Some screen readers may fail to tell the user whether the print-end chat menu is open or not
+help.webchat.howaccessible.li9=Jaws stops reading out the chat conversation when the user is in the chat textbox. Users should click ‘tab button’ and click up arrow
+help.webchat.howaccessible.li10=Users of Jaws may find they receive no notification that the agent has sent a message
+
+help.webchat.reportingproblems.h2=Reporting accessibility problems with this webchat
+help.webchat.reportingproblems.p1=We’re always looking to improve the accessibility of the webchat. If you find any problems not listed on this page or think we’re not meeting accessibility requirements, contact {0}
+help.webchat.reportingproblems.email=mailto:ash.chadwick@hmrc.gov.uk
```
Comment (oc-ste) [inline conf/messages]: should not have href logic in messages

File: conf/messages [hunk 4]
```diff
@@ -200,3 +200,59 @@ language-switcher.welsh=Cymraeg
...
+help.webchat.howaccessible.li6=Users may find that when the chat is finished, they cannot tab backwards through the chat
+help.webchat.howaccessible.li7=To find the privacy and accessibility links, users should tab beyond the message input box
+help.webchat.howaccessible.li8=Some screen readers may fail to tell the user whether the print-end chat menu is open or not
+help.webchat.howaccessible.li9=Jaws stops reading out the chat conversation when the user is in the chat textbox. Users should click ‘tab button’ and click up arrow
+help.webchat.howaccessible.li10=Users of Jaws may find they receive no notification that the agent has sent a message
+
+help.webchat.reportingproblems.h2=Reporting accessibility problems with this webchat
+help.webchat.reportingproblems.p1=We’re always looking to improve the accessibility of the webchat. If you find any problems not listed on this page or think we’re not meeting accessibility requirements, contact
+help.webchat.reportingproblems.emailtext=ash.chadwick@hmrc.gov.uk
```
Comment (oc-ste) [inline conf/messages]: still need this? It's reportingProblemsEmail

**Comment by daniel-manning** [comment]
pages should use messages in preference to Messages but as this is from a fork I'll merge and correct

**Pull request merged**